pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3076,7 +3076,7 @@ func (c *Controller) syncVMAnnotationsToVMI(vm *virtv1.VirtualMachine, vmi *virt
 
 	annotationsToSync := []string{
 		descheduler.EvictPodAnnotationKeyAlpha,
-		descheduler.EvictPodAnnotationKeyBeta,
```

Comment:
Can we keep the old one if out there exists a verion that still supports it?

Reply:
The PR with `descheduler.beta.kubernetes.io/evict` never got merged upstream, see:
https://github.com/kubernetes-sigs/descheduler/pull/1673

Reply:
Thanks